src/components/Settings/SettingsMain.tsx

```diff
@@ -48,6 +48,7 @@ const messages = defineMessages({
   validationApplicationTitle: 'You must provide an application title',
   validationApplicationUrl: 'You must provide a valid URL',
   validationApplicationUrlTrailingSlash: 'URL must not end in a trailing slash',
+  originalLanguageDefault: 'All Languages',
```

Comment:
I think you need to do this for the user settings page also.

Reply:
Oh right, I forgot about that!  Fixing now